db/schema.rb

```diff
@@ -13,7 +13,7 @@
 
 ActiveRecord::Schema.define(version: 20141005040238) do
 
-  create_table "widgets", force: true do |t|
+  create_table "widgets", force: :cascade do |t|
```

Comment:
It was automatic, but I did notice it and looked it up. It's actually the first thing listed on the notable changes for 4.2.0.
http://guides.rubyonrails.org/4_2_release_notes.html#active-record-notable-changes